kubernetes/README.md

```diff
@@ -118,3 +128,59 @@ $ VM_NAME=xXx \
 ```
 
 > NOTE: Both kubernetes playbooks (`k8s` and `k8s-misc`) can have it's host overriden using the `override_host` variable, e.g., `-e override_host=localhost` to launch it on the controller itself.
+
+# Troublehsooting
+
+## Expired k8s certificate
+
+### Problem
+
+- kubelet service shows connection errors. 
+- The docker container running the k8s API server cannot be started
+
+### Cause
+
+```bash
+$ docker logs xxxxxxxxxxxx
+...
+W0121 11:09:31.447982       1 clientconn.go:1251] grpc: addrConn.createTransport failed to connect to {127.0.0.1:2379 0  <nil>}. Err :connection error: desc = "transport: authentication handshake failed: x509: certificate has expired or is not yet valid". Reconnecting...
+```
+
```

Comment:
Add. Check the validity of the kubernetes certificate using the following command. If they have been expired, then apply the trick as defined at the section `Solution`